iso15118/shared/comm_session.py

```diff
@@ -391,8 +392,10 @@ async def stop(self, reason: str):
         logger.info(f"Reason: {reason}")
 
         await asyncio.sleep(2)
-        # TODO Signal data link layer to either terminate or pause the data
-        #      link connection
+        # Signal data link layer to either terminate or pause the data
+        # link connection
+        if hasattr(self.comm_session, "evse_controller"):
+            await self.comm_session.evse_controller.session_stop(terminate_or_pause)
```

Comment:
To help test this feature (specifically pause), I'm thinking we could add a similar session_stop() on evcc side as well? If so, this would be modified to include an else condition for ev_controller as well.
```
if hasattr(self.comm_session, "evse_controller"):
  await self.comm_session.evse_controller.session_stop(terminate_or_pause)
elif hasattr(self.comm_session, "ev_controller"):
  await self.comm_session.ev_controller.session_stop(terminate_or_pause)
```
What do you think?

Reply:
As we talked, no need for ev_controller for now.